Exiled.API/Features/Pickups/Scp244Pickup.cs

```diff
@@ -0,0 +1,97 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp244Pickup.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Pickups
+{
+    using Exiled.API.Features.DamageHandlers;
+    using Exiled.API.Features.Items;
+
+    using InventorySystem.Items.Usables.Scp244;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for SCP-330 bags.
+    /// </summary>
+    public class Scp244Pickup : Pickup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Scp244Pickup"/> class.
+        /// </summary>
+        /// <param name="itemBase">The base <see cref="Scp244DeployablePickup"/> class.</param>
+        public Scp244Pickup(Scp244DeployablePickup itemBase)
+            : base(itemBase)
+        {
+            Base = itemBase;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Scp244DeployablePickup"/> that this class is encapsulating.
+        /// </summary>
+        public new Scp244DeployablePickup Base { get; }
+
+        /// <summary>
+        /// Gets the speed of Scp244's too grow.
+        /// </summary>
+        public float GrowSpeed => Base.GrowSpeed;
+
+        /// <summary>
+        /// Gets the time for the sphere to finish their expansion.
+        /// </summary>
+        public float TimeToGrow => Base.TimeToGrow;
+
+        /// <summary>
+        /// Gets the current size effect of the Scp244's Hypothermia.
+        /// </summary>
+        public float CurrentDiameter => Base.CurrentDiameter;
+
+        /// <summary>
+        /// Gets or sets the current size percent of the Scp244's Hypothermia.
+        /// </summary>
+        public float CurrentSizePercent
+        {
+            get => Base.CurrentSizePercent;
+            set => Base.CurrentSizePercent = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the Scp244's remaining health.
+        /// </summary>
+        public float Health
+        {
+            get => Base._health;
+            set => Base._health = value;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether or not this Scp244 is breakable.
+        /// </summary>
+        public bool IsBreakable => Base.State == Scp244State.Idle || Base.State == Scp244State.Active;
```

Comment:
`Base.State is Scp244State.Idle or Scp244State.Active`

Reply:
true